src/openrct2-ui/interface/Window.cpp

```diff
@@ -263,21 +233,16 @@ rct_window* window_create_auto_pos(
         if (w->flags & WF_STICK_TO_BACK)
             continue;
 
-        screenPos = { w->windowPos.x + w->width + 2, w->windowPos.y };
-        if (window_fits_on_screen(screenPos, width, height))
-            goto foundSpace;
-
-        screenPos = { w->windowPos.x - w->width - 2, w->windowPos.y };
-        if (window_fits_on_screen(screenPos, width, height))
-            goto foundSpace;
-
-        screenPos = { w->windowPos.x, w->windowPos.y + w->height + 2 };
-        if (window_fits_on_screen(screenPos, width, height))
-            goto foundSpace;
+        std::array offsets = { ScreenCoordsXY{ w->width + 2, 0 }, ScreenCoordsXY{ -w->width - 2, 0 },
```

Comment:
Const

Reply:
Done. I assume you'd want 4-line array in here as well. I don't know how to fool `clang-format` in this case, as I think comments are not useful. I can turn it off and on, but not sure this is desirable

Reply:
yeah it would be preferred. I don't see any harm turning it off and on if we cant get what we want.

Reply:
Alright, done